src/meshpy/core/nurbs_patch.py

```diff
@@ -66,33 +66,36 @@ def get_nurbs_dimension(self):
             )
         return n_knots
 
+    def get_number_of_control_points_per_dir(self):
+        """Return the number of control points in each parameter direction of
+        the patch."""
```

Comment:
Could you fix the docstring and type hints (for the return value) here as well?

Reply:
Sure! Return type is clear to me, but how do you think the doctoring should look like?

Reply:
The `Returns:` is missing

Maybe something along the lines

```suggestion
    def get_number_of_control_points_per_dir(self):
        """Determine the number of control points in each parameter direction of
        the patch.
        
        Returns:
            List of control points per direction"""
```

Reply:
Good point, to be honest, the return doc string has usually been ignored by me 😬. But let's do this going further!